recipe/meta.yaml

```diff
@@ -1,25 +1,20 @@
 package:
   name: conda-forge-build-setup
-  version: 1.1.0
+  version: 2.0.0
 
 build:
   number: 0
   script:
-    - IF NOT EXIST %PREFIX%\etc\conda\activate.d MKDIR %PREFIX%\etc\conda\activate.d  # [win]
-    - COPY %RECIPE_DIR%\activate.bat %PREFIX%\etc\conda\activate.d\conda-forge-build-setup_activate.bat  # [win]
-    - mkdir -p "${PREFIX}/etc/conda/activate.d"  # [unix]
-    - cp "${RECIPE_DIR}/activate.sh" "${PREFIX}/etc/conda/activate.d/conda-forge-build-setup_activate.sh"  # [unix]
+    - if not exist "%LIBRARY_BIN%" mkdir "%LIBRARY_BIN%"                                                                 # [win]
+    - mkdir -p "${PREFIX}/bin"                                                                                           # [unix]
+    - COPY "%RECIPE_DIR%\\run_conda_forge_build_setup_win" "%LIBRARY_BIN%\\run_conda_forge_build_setup.bat"              # [win]
+    - cp "${RECIPE_DIR}/run_conda_forge_build_setup_osx" "${PREFIX}/bin/run_conda_forge_build_setup"                     # [osx]
+    - cp "${RECIPE_DIR}/run_conda_forge_build_setup_linux" "${PREFIX}/bin/run_conda_forge_build_setup"                   # [linux]
 
-requirements:
-  run:
-    - conda-build         # [linux]
-    - conda-build 1.20.0  # [osx]
-    - conda-build 1.20.0  # [win64 and py34]
-    - conda-build         # [win and not (win64 and py34)]
-    - jinja2
-    - anaconda-client
-
-test: {}
+test:
+  commands:
+    - if not exist "%LIBRARY_BIN%\\run_conda_forge_build_setup.bat" exit 1                                               # [win]
```

Comment:
Or simply add a small test script to the travis.yml/... files? I guess it's also better not to use the "normal" conda-smithy script here in case the scripts in the old version render the the feedstock unuseable?